src/index.js

```diff
@@ -20,23 +21,40 @@ import output from './output';
  * This is adapter function that will send the data to the chosen output module
  * @param {*} data 
  */
-const sendToOutput = (...data) => {
+const sendToOutput = (data) => {
```

Comment:
Let's take it out and make index as small as possible, all handling will be done in output maybe we can put an additional comment that comes from command

Reply:
👍